alpine-base-redis/Dockerfile.tmpl

```diff
@@ -1,6 +1,6 @@
 FROM unocha/alpine-base-s6:%%UPSTREAM%%
 
-MAINTAINER "Serban TEODORESCU <teodorescu.serban@gmail.com>"
+MAINTAINER UN-OCHA Operations <ops+docker@humanitarianresponse.info>
```

Comment:
I'm so confused about humanitarianresponse.info vs reliefweb.int.

Reply:
Legacy vs current. This is just a copy & paste job. Doesn't matter that much anyway, we never look at it or use it for anything ;-)

Reply:
I think `MAINTAINER` is deprecated.

Reply:
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated